test/test_truncate_table.py

```diff
@@ -0,0 +1,9 @@
+from sql_metadata import Parser
+from sql_metadata.keywords_lists import QueryType
+
+
+def test_truncate_table():
+    parser = Parser("TRUNCATE TABLE foo")
+    assert parser.query_type == QueryType.TRUNCATE
+    assert parser.tables == ["foo"]
+    assert parser.columns == []
```

Comment:
```suggestion
    assert parser.columns == []

```

Adding a new line to make `black` happy.

Reply:
@T-Santos, can you commit the above suggestion?